File: 11531.py
logs = {}
cnt = 0; penalty = 0
while 1:
    s = input()
    if s[:2] == '-1': break

    m, name, correct = s.split()
    m = int(m)

    if correct == 'right':
        cnt += 1
        if name in logs:
            penalty += m + logs[name] * 20
        else:
            penalty += m
    else:
        if name in logs:
            logs[name] += 1
        else:
            logs[name] = 1

print(cnt, penalty)

File: 11531.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
	vector<pair<char, int>> logs;
	int cnt = 0, penalty = 0;
	while (1) {
		int m; cin >> m;
		if (m == -1) break;

		char name; string correct;
		cin >> name >> correct;

		int index = -1;
		for (int i = 0; i < logs.size(); i++) {
			if (logs[i].first == name) {
				index = i;
				break;
			}
		}

		if (correct == "right") {
			cnt++;
			if (index != -1)
				penalty += (m + logs[index].second * 20);
			else
				penalty += m;
		}
		else {
			if (index == -1)
				logs.push_back(pair<char, int>(name, 1));
			else
				logs[index].second++;
		}
	}

	cout << cnt << ' ' << penalty << endl;
}
